### PR TITLE
[omnibus] strip symbols

### PR DESCRIPTION
**Pull request by truthbk**
### What does this PR do?

This PR strips the symbols to the binaries/libs on linux. 

The current problem with the PR is these symbols are currently deleted after being stripped, we don't want to do this, we want to zip them up and upload them somewhere. So that's pending.  

### Motivation

Decrease package size.

### Additional Notes

Please make sure we save the symbols somewhere.

Sister PR: https://github.com/DataDog/omnibus-ruby/pull/91 (all heavy lifting done there)

### CODE REVIEW

**Review by KSerrania — CHANGES_REQUESTED** [review]
As I mentioned in the related [omnibus-ruby PR](https://github.com/DataDog/omnibus-ruby/pull/91), the `.gitlab-ci.yml` file also needs to be updated:
- [this line](https://github.com/DataDog/datadog-agent/blob/667605285a2f65d9bd5c5281b5d13be97a6a9bdb/.gitlab-ci.yml#L354) fails in the Debian build because now two files match the expression, and the `-c` option can only take one,
- [this line](https://github.com/DataDog/datadog-agent/blob/667605285a2f65d9bd5c5281b5d13be97a6a9bdb/.gitlab-ci.yml#L417) of the RPM build fails because it tries to install the debug package, which fails because it's missing a dependency (the main package).

File: omnibus/config/projects/agent.rb [hunk 1]
```diff
@@ -191,3 +191,8 @@
 
 exclude '\.git*'
 exclude 'bundler\/git'
+
+if linux?
+  strip_build true
+  debug_path ".debug"  # the srip symbols will be in here
```
Comment (albertvaka) [inline omnibus/config/projects/agent.rb]: You don't reference this `.debug` directory anywhere else? Just making sure this is intended.

**Review by albertvaka — DISMISSED** [review]

**Comment by truthbk** [comment]
This sister PR will also need to be merged before this one: https://github.com/DataDog/omnibus-ruby/pull/94

File: omnibus/config/projects/agent.rb (hunk 1 above)
Comment (truthbk) [reply omnibus/config/projects/agent.rb]: Yes, this is intended, the `.debug` path is created by the stripper by `omnibus-ruby`. I'll add a clarifying note.

**Review by KSerrania — APPROVED** [review]
LGTM, there's a problem with the package sizes of the `deb` packages, but it has to do with the 
`omnibus-ruby` PR.

**Pull request merged**